docs/concepts/what-is-a-codec-server.md

```diff
@@ -0,0 +1,35 @@
+---
+id: what-is-a-codec-server
+title: What is a Codec Server?
+sidebar_label: Codec Server
+description: Explanation and implementation of a remote encryption/decryption server.
+---
+
+A Codec Server is an optional implementation for your [Temporal Platform](/docs/concepts/what-is-the-temporal-platform). With it, deserialized Payloads can have their data displayed on your WebUI.
```

Comment:
Maybe something like...

"A Codec Server is an optional feature used to encrypt and decrypt data payloads for the Web UI."